services/base-service/models/expert/api.js

```diff
@@ -4,6 +4,19 @@ const ExpertModel = require('./model.js');
 const {defaultEsApiGenerator} = dataModels;
 const md5 = require('md5');
 
+// Custom middleware to check Content-Type
+function json_only(req, res, next) {
```

Comment:
@UcDust 
`Accept:` not `Content-type:` and should this check that application/json is one of the types it will accept